move:
  ros__parameters:
    plugins:
      - plansys2_move_bt_node
    waypoints: ["wheels_zone", "steering_wheels_zone", "body_car_zone", "assembly_zone", "recharge_zone"]
    waypoint_coords:
      wheels_zone: [0.0, -2.0, 0.0]
      steering_wheels_zone: [1.8, 0.0, 0.0]
      body_car_zone: [0.0, 2.0, 0.0]
      assembly_zone: [-2.0, -0.4, 0.0]
      recharge_zone: [-2.0, -0.4, 0.0]

prepick:
  ros__parameters:
    plugins:
      - plansys2_open_gripper_bt_node
      - plansys2_set_gripper_pos_bt_node
pick:
  ros__parameters:
    plugins:
      - plansys2_pick_bt_node

prerelease:
  ros__parameters:
    plugins:
      - plansys2_set_gripper_pos_bt_node
release:
  ros__parameters:
    plugins:
      - plansys2_release_bt_node
